Port 10849 documentation for azure devops boards (#1656)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/azure-devops/examples.md

- Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/wit/work-items/list?view=azure-devops-rest-7.1&tabs=HTTP#get-list-of-work-items) for the Azure Devops work item object structure.
- Click [Here](https://learn.microsoft.com/en-us/azure/devops/boards/queries/wiql-syntax?view=azure-devops#example-clauses) for WIQL clauses
:::

## Mapping boards

> In the following example, you'll learn how to ingest Azure DevOps Boards into Port.  You may use the following Port blueprint definitions and integration configuration: 

Reviewer: ```suggestion
The example below shows how to ingest Azure DevOps Boards into Port.  
You can use the following Port blueprint definitions and integration configuration: 
```

- Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/wit/work-items/list?view=azure-devops-rest-7.1&tabs=HTTP#get-list-of-work-items) for the Azure Devops work item object structure.
- Click [Here](https://learn.microsoft.com/en-us/azure/devops/boards/queries/wiql-syntax?view=azure-devops#example-clauses) for WIQL clauses
:::

## Mapping boards

> In the following example, you'll learn how to ingest Azure DevOps Boards into Port.  you may use the following Port blueprint definitions and integration configuration: 

Reviewer 2: ```suggestion
In the following example, you'll learn how to ingest Azure DevOps Boards into Port.  You may use the following Port blueprint definitions and integration configuration: 
```

> <ProjectBlueprint/>

Reviewer 2: Let's remove this as well

<BoardBlueprint/>

<PortBoardAppConfig/>

:::tip To Learn More
- Refer to the [setup](azure-devops.md#setup) section to learn more about the integration configuration setup process.

> - We leverage [JQ JSON processor](https://stedolan.github.io/jq/manual/) to map and transform Azure DevOps objects to Port entities.

Reviewer 2: Remove these items, since it's repeated in the other examples
